app/assets/stylesheets/nfg_ui/network_for_good/admin/application.scss

```diff
@@ -21,3 +21,8 @@
 
 // Legacy browser support
 @import 'nfg_ui/network_for_good/public/legacy_browser_support/application';
+
+.disabled, [disabled] {
+  pointer-events: none !important;
+  cursor: not-allowed !important;
+}
```

Comment:
Let's move this to the `/app/assets/stylesheets/nfg_ui/network_for_good/public/nfg_theme/_reboot.scss` file at the bottom. There's a similar disabled style listed there already.